tests/unit/standalone/test_slurm_command.py

```
@@ -55,7 +55,8 @@ def test_generate_slurm_command(self):
             exclusive=True,
             openmpi_oversubscribe=True,
             slurm_cmd_args=["--help"],
+            run_time_limit=250,
         )
-        self.assertEqual(len(command_lst), 12)
-        reply_lst = ['srun', '-n', '1', '-D', '/tmp/test', '-N', '1', '--cpus-per-task=2', '--gpus-per-task=1', '--exact', '--oversubscribe', '--help']
+        self.assertEqual(len(command_lst), 13)
+        reply_lst = ['srun', '-n', '1', '-D', '/tmp/test', '-N', '1', '--cpus-per-task=2', '--gpus-per-task=1', '--exact', '--oversubscribe', '--time=5', '--help']
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)